servicetalk-transport-api/src/main/java/io/servicetalk/transport/api/TransportObservers.java

```diff
@@ -38,4 +38,16 @@ public static TransportObserver asSafeObserver(final TransportObserver observer)
         }
         return new CatchAllTransportObserver(observer);
     }
+
+    /**
+     * Combines two {@link TransportObserver}s into a single {@link TransportObserver}.
+     *
+     * @param first the {@link TransportObserver} that will receive events first
+     * @param second the {@link TransportObserver} that will receive events second
+     * @return a {@link TransportObserver} that delegates all invocations to the {@code first} and {@code second}
+     * {@link TransportObserver}s
+     */
+    public static TransportObserver biTransportObserver(final TransportObserver first, final TransportObserver second) {
```

Comment:
For a public API, I would prefer a `combine()` method with a var-arg and if we want, we can optimize for 2 vs many cases.

Reply:
Good idea! Added support for multiple